sale_order_line_tag/views/sale_order_views.xml

```diff
@@ -55,9 +55,9 @@
     <menuitem
         id="sale_order_line_tag_action_menu"
         name="Sales Order Line Tags"
-        parent="sale.menu_sale_config"
+        parent="sale.menu_sales_config"
         action="sale_order_line_tag_action"
-        sequence="25"
+        sequence="20"
```

Comment:
@pmonje-duonex Why changing this ?

Reply:
This change was suggested by me. Once this menu was moved to "Sales Order" submenu, I've inspected existing submenus for `sale` and `sale_management` addons and sequence starts from `1` sequence for Quotation Templates and continues with Sales tags (`10`)... and then simply proposed `20`. There's no problem keeping `25` IMO,

Reply:
No it's ok, Simple question